### PR TITLE
[AC-5059] - Create a Boilerplate Django-docker project

### PR DESCRIPTION
**Pull request by yotammanor**
**Changes Introduced:** 
This was created with the intent of serving Accelerator-Directory's MVP as is required by [AC-5059](https://masschallenge.atlassian.net/browse/AC-5059), and is a cleaned up and slightly adjusted version of impact-api's settings. 

Since it is a byproduct of the original ticket, some configuration tasks were left more manual then is reasonable, but this is a follow on task described [here](https://masschallenge.atlassian.net/browse/AC-5091).

**Test:**

- Go through the instructions in the Readme.md file.
- make sure you're able to see a hello-world page. (note that the default suggested configuration is used by accelerator-directory. so to avoid conflict, remove it, or better yet - change it and show that all three can be used simultaneously).
- Review + groom follow on ticket https://masschallenge.atlassian.net/browse/AC-5091

**Note:**
- wasn't sure how to structure the branches. Went for a weird solution of merging directly to branch so all changes are visible. feel free to close and reopen or change as needed.

### CODE REVIEW

File: web/directory/static-compiled/admin/css/base.css [hunk 1]
```diff
@@ -0,0 +1,978 @@
+/*
```
Comment (mc-nathan) [inline web/directory/static-compiled/admin/css/base.css:1]: Not a blocker, but I have to wonder if it is necessary to have the stuff in web/directory/static-compiled checked into the repo?  I far rather see this come from somewhere else.  128 files is pretty big for a "boilerplate".

File: web/directory/static/css/dev/style.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
+/* line 11, ../../sass/style.scss */
```
Comment (mc-nathan) [inline web/directory/static/css/dev/style.css:1]: Not a blocker, but web/directory/static has a lot of duplication with web/directory/static-compiled that I'd like to see removed from the repo.  If they both need to exist for some reason, then there should be one version checked in and the other should be copied as part of the `make build` process.

**Review by mc-nathan — APPROVED** [review]
This is not really part of the definition of done for AC-5059, but is useful start on something we want to keep.  There is a follow on ticket, so I'm OK with this as it is, but there are definitely some things that we should plan on improving.

File: web/directory/static-compiled/admin/css/base.css (hunk 1 above)
Comment (yotammanor) [reply web/directory/static-compiled/admin/css/base.css:1]: Honestly I'm surprised this is not git-ignored by default in impact-api. But I didn't want to spend time figuring it out, so I just included it. Generally speaking, I expect that if @shanbady goes over this Repo he'll be able to make quick clean up gains.

File: web/directory/static/css/dev/style.css (hunk 2 above)
Comment (yotammanor) [reply web/directory/static/css/dev/style.css:1]: See comment above.

File: web/directory/static-compiled/admin/css/base.css (hunk 1 above)
Comment (pbx) [reply web/directory/static-compiled/admin/css/base.css:1]: I agree that generated/compiled/built files should not be committed to the repo. I added a note to this effect on the follow-on ticket AC-5091, just to remind us.

**Review by pbx — APPROVED** [review]
I approve of this initiative. I expect in addition to refining this in the follow-on AC-5091, we'll find things we want to work on when we spin up our next standalone component.

**Pull request merged**